connectors/src/connectors/notion/index.ts

```diff
@@ -601,12 +601,16 @@ export class NotionConnectorManager extends BaseConnectorManager<null> {
       return new Err(new Error("Connector not found"));
     }
 
+    const id = internalId.startsWith("notion-")
+      ? internalId.substring("notion-".length)
+      : internalId;
+
     const memo = memoizationKey || uuidv4();
 
     try {
       const parents = await getParents(
         this.connectorId,
-        internalId,
+        id,
```

Comment:
Isn't this a breaking change with what we have today? This `getParents` function relies on a Redis cache, what ids do we store there?

Reply:
No, the only difference is that we support id starting with "notion-" (used only in tables) - if the id is a "standard" notion id it's the same value

Reply:
So databases (tables) in Notion have an id that starts with `notion-` while for documents no? If that's the case then IMHO it's worth a comment.

Reply:
> So databases (tables) in Notion have an id that starts with `notion-` while for documents no? If that's the case then IMHO it's worth a comment.

we can forget about it, it was only a hack for cli usage with notion-* ids, but does make sense anyway